### PR TITLE
Removed the redundant version

### PR DESCRIPTION
**Pull request by arpit7714**
removed the redundant version
of the setuptools from 21.0.0 to 21

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by nityeshaga** [comment]
These are the problems with your commit message:
* the short description (the part after `:` on the first line) needs to start with a capital letter
* the short description needs to *describe* the change instead of just saying 

> made changes

* its also better if you make the commit body complete grammar sentences like they are in the guide (starts with capital letter, ends in a punctuation)

All the best!

File: coalib/coala_modes.py [hunk 1]
```diff
@@ -26,7 +26,8 @@ def mode_normal(console_printer, log_printer, args, debug=False):
         console_printer=console_printer,
         args=args,
         debug=debug)
-
+    if log_printer:
```
Comment (nityeshaga) [inline coalib/coala_modes.py]: Does this change really belong to this issue?
Commits should be atomic and no one commit should not deal with multiple changes.

Comment (arpit7714) [reply coalib/coala_modes.py]: no ! mistakenly i do that actually i was working in same branch for two different issue h

Comment (arpit7714) [reply coalib/coala_modes.py]: how to delete that commit ??

Comment (nityeshaga) [reply coalib/coala_modes.py]: Both the changes are in one commit only. So you could try to make relevant changes to the file (remove those extra lines) and save the file. After you are done with it, do `git add <filename>`. Then `git commit --amend` and save it. Then push it with `-f` option.

**Comment by nityeshaga** [comment]
You got me wrong :sweat_smile: You didn't have to change the PR title. You need to change the commit message.

File: coalib/coala_modes.py (hunk 1 above)
Comment (nityeshaga) [reply coalib/coala_modes.py]: Also when you do `git --amend` you will get an option to change the commit message. You should make the changes that I suggested (for the commit message) then.

**Comment by monsij** [comment]
@arpit7714 If you are facing issues making proper commit messages ,do have a look at the newcomer's guide.Additionally you can also take a look at previous commits which have successfully passes the tests!

**Comment by jayvdb** [comment]
Gah.  Five commits on https://github.com/coala/coala/pull/5183/commits

Please fix *this* PR to have only one commit.  Do *NOT* create a new PR.  Fix this one.

**Comment by Makman2** [comment]
Closing due to inactivity.

**Pull request closed without merging**